### PR TITLE
BaSP-MR-114: Admins move get and delete logic 

### PR DESCRIPTION
**Pull request by ale-schmidt**
Replace the data from the local state.

Move the logic of the get to the thunk file in the admins folder and apply it to the entity index file.

Move the logic of the delete to the thunk file in the admins folder and apply it to the entity index file.

To make the user experience more pleasant, for each request, a spinner (isLoading) should be implemented so that the user understands that a request is being made, for example when loading the table.

### CODE REVIEW

File: src/Redux/admins/thunks.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {
+  getAdminsPending,
+  getAdminsSuccess,
+  getAdminsError,
+  deleteAdminPending,
+  deleteAdminSuccess,
+  deleteAdminError
+} from './actions';
+
+export const getAdmins = () => {
```
Comment (EmaBaldor) [inline src/Redux/admins/thunks.js:13]: You should also add the getAdminById.

**Review by marcohzo — APPROVED** [review]
Everything looks great!

**Comment by Federico-Peralta** [comment]
[[Admins] - Move get and delete logic](https://trello.com/c/W3SyecSa/114-admins-move-get-and-delete-logic)

**Comment by ale-schmidt** [comment]
[[Admins] - Move get and delete logic](https://trello.com/c/W3SyecSa/114-admins-move-get-and-delete-logic)

File: src/Redux/admins/thunks.js (hunk 1 above)
Comment (ale-schmidt) [reply src/Redux/admins/thunks.js:13]: amazing, was done!

File: src/Redux/admins/reducer.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import {
+  GET_ADMINS_PENDING,
+  GET_ADMINS_SUCCESS,
+  GET_ADMINS_ERROR,
+  GET_ADMINBYID_PENDING,
```
Comment (EmaBaldor) [inline src/Redux/admins/reducer.js:5]: Note that this type of constant must respect the 'GET_ADMINS_BY_ID_PENDING' convention.

**Review by EmaBaldor — COMMENTED** [review]
Good Job!

**Review by EmaBaldor — APPROVED** [review]

**Comment by araefedg** [comment]
The implementation works correctly, very good Ale!

**Comment by araefe1992** [comment]
Everything is working properly as expected!

**Pull request merged**